[mypy-xtb-service_version]
ignore_errors = True

[mypy-versioneer]
ignore_errors = True
